src/main/java/com/google/api/codegen/util/php/PhpPackageUtil.java

```diff
@@ -62,4 +50,40 @@ public static String getFullyQualifiedName(String packageName, String objectName
   public static boolean isPackageVersion(String versionString) {
     return versionString.matches(PACKAGE_VERSION_REGEX);
   }
+
+  /**
+   * Remove the base package name, returning a package name which begins with the version. If no
+   * version is present in the input packageName, returns null.
+   */
+  public static String removeBasePackageName(String packageName) {
+    ArrayList<String> packageComponents = new ArrayList<>();
+    List<String> pieces = Arrays.asList(PhpPackageUtil.splitPackageName(packageName));
+    boolean foundVersion = false;
+    for (String packageElement : Lists.reverse(pieces)) {
+      packageComponents.add(packageElement);
+      if (isPackageVersion(packageElement)) {
+        foundVersion = true;
```

Comment:
This looks redundant. You can just do `return buildPackageName(Lists.reverse(packageComponents));` here and `return null` at the end of the method (no `if` statement or `foundVersion` variable needed)

Reply:
Fixed